platform-demo-module/src/main/java/com/softicar/platform/demo/module/moment/DemoMomentPointInTimeInput.java

```diff
@@ -0,0 +1,21 @@
+package com.softicar.platform.demo.module.moment;
+
+import com.softicar.platform.emf.attribute.field.daytime.EmfDayTimeInput;
+
+public class DemoMomentPointInTimeInput extends EmfDayTimeInput {
+
+	private final AGDemoMoment moment;
+
+	public DemoMomentPointInTimeInput(AGDemoMoment moment) {
+
+		this.moment = moment;
+	}
+
+	@Override
+	public void refreshInputConstraints() {
```

Comment:
- not sure if this test shows a realistic example, since changing the input value on `refreshInputConstraints` is quite unexpected, any ideas?